code/components/citizen-server-impl/include/state/ServerGameState.h

```diff
@@ -508,6 +508,16 @@ class ServerGameState : public fwRefCountable, public fx::IAttached<fx::ServerIn
 		m_entityLockdownMode = mode;
 	}
 
+	inline float GetEntityScopeRadius()
+	{
+		return m_entityScopeRadius;
+	}
+
+	inline void SetEntityScopeRadius(float scopeRadius)
```

Comment:
Is this meant to be exposed by a native?

Reply:
if ever a 'global' radius can be set, yeah why not